src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicencePolicyService.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.AdditionalConditionData
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.Licence
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.policy.LicencePolicy
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.policy.PolicyChanges
+import javax.persistence.EntityNotFoundException
+
+data class ConditionChanges<T>(
+  val removedConditions: List<T>,
+  val addedConditions: List<T>,
+  val amendedConditions: List<Pair<T, T>>
+)
+
+data class ConditionChangesByType<T, U>(
+  val Ap: ConditionChanges<T>,
+  val Pss: ConditionChanges<U>
+)
+
+data class LicenceConditionChanges(
+  val code: String,
+  val sequence: Int?,
+  val previousText: String,
+  val currentText: String,
+  var dataChanges: List<AdditionalConditionData>
+)
+
+class LicencePolicyService(private val policies: List<LicencePolicy>) {
+
+  fun currentPolicy(): LicencePolicy = policies.maxBy { it.version }
```

Comment:
Do we want to have a safeguard of locking this to v2.0 for now? I can put it in the front-end instead if you like?
Or are we just going to not release anything until Monday?

Reply:
probably not release till monday - should be easy to revert this if necessary